src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -141,80 +142,71 @@ const ChartsHolder = (props: {
   const recommendationsFedModalCopy = getModalCopyWithFedLevel(
     projection,
     projection.locationName,
-    props.projections.getMetricValues(),
+    projections.getMetricValues(),
   );
 
   const recommendationsHarvardModalCopy = getModalCopyWithHarvardLevel(
     projection,
     projection.locationName,
-    props.projections.getMetricValues(),
+    projections.getMetricValues(),
   );
 
   // TODO(pablo): Create separate refs for signup and share
   return (
     <>
-      {!projection ? (
-        <NoCountyDetail
-          countyId={props.county?.county_url_name}
-          stateId={props.stateId}
```

Comment:
I removed this because projection is never null here.  I may re-add it back though just for the sake of the diff

Reply:
re-added it :D